package.json

```diff
@@ -1,8 +1,20 @@
 {
+  "engines": {
+    "node": ">=17.0.0"
```

Comment:
Probably worth declaring `^` here

Reply:
Depends. Our container used alpine latest and thus I guess it always contains the very latest version of node. Is that captured by `^`? Not sure if the constraints are the same as for composer 🤷🏼‍♂️

Reply:
Yeah, those are SemVer, so IMO we should pin to a major

Reply:
Do you have an idea on how to keep `package.json` and `Dockerfile` in sync then?
Because the main idea here was to always ensure that we do run on latest but with a minimum of `17.0.0`.
That `engines` config is parsed by `eslint` and thats why I've added it.

Is this something we could use renovate for?

Reply:
Changed to `^17` along with using `node:17-alpine` in the `Dockerfile`.